pyaedt/modules/AdvancedPostProcessing.py

```diff
@@ -473,7 +471,7 @@ def plot_field(
         ----------
         quantity : str
             Quantity to plot (e.g. ``"Mag_E"``).
```

Comment:
```suggestion
            Quantity to plot. For example, ``"Mag_E"``.
```
Avoid Latin phrases.